<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smart_home</class>
 <widget class="QWidget" name="smart_home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>smart_home</string>
  </property>
  <widget class="QLabel" name="IP_L">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;IP：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IP">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Port_L">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Port：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Port">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="connect">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnect">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="record">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Video">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="smart_home.qrc">:/jpg/camera.jpg</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Temp_L">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="TEM_Unit_L">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>120</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>°C</string>
   </property>
  </widget>
  <widget class="QLabel" name="HUM_L">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Humidity</string>
   </property>
  </widget>
  <widget class="QLabel" name="HUM_Unit_L">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>%RH</string>
   </property>
  </widget>
  <widget class="QLabel" name="AC_L">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Guard</string>
   </property>
  </widget>
  <widget class="QLabel" name="Light">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLCDNumber" name="TEM_M">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="HUM_M">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="AtHome">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>120</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>At home</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="LeaveHome">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>150</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Out home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Checked">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>210</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Checked</string>
   </property>
  </widget>
  <widget class="QLabel" name="fired">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>120</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Aharoni</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>*fired*</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="smart_home.qrc"/>
 </resources>
 <connections/>
</ui>
